archinstall/lib/user_interaction.py

```diff
@@ -579,6 +579,7 @@ def select_profile():
 
 		print(' -- The above list is a set of pre-programmed profiles. --')
 		print(' -- They might make it easier to install things like desktop environments. --')
+		print(' -- The desktop profile will let you select a DE/WM profile, e.g sway,kde plasma and gnome --')
```

Comment:
This is fine for now, but in the long run I would like to get a shorter description of each profiles via `__description__` or a top-in-file docstring. And have it present like:
```
0: desktop - Lets you select a desktop environment like kde or gnome
1: server - ...
```
But that's after release party! : )

Reply:
Started this already in #520. Also I made a slight change to this in #519 with rationale in the PR.